### PR TITLE
New order modal

### PR DESCRIPTION
**Pull request by RivkaEhrenfeld**
new order modal

### CODE REVIEW

File: src/apiCalls/orderCalls.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import {IOrder} from "../interfaces/IOrder"
+import { BASE_URL } from "../config/config";
+
+export const addNewOrder=async(order:IOrder)=>{
+
+    
+    const token = localStorage.getItem("token");
+    const req = {
+                method: 'POST',
+                headers: { 'Content-Type': 'application/json', 'token': token != null ? token : "" },
```
Comment (efef10) [inline src/apiCalls/orderCalls.ts:11]: 1. no need to add token for each request, should be done globally on axios interceptor
2. no need to JSON.stringify the request, send it as plain object

File: src/components/MyAutocomplete.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    getFunction: (prefix: string) => Promise<{ [key: string]: any; }[]>
+    setItem: any
+    displayField: number
+}
+
+
+function MyAutocomplete(props: prp) {
+    const [data, setData] = useState([] as { [key: string]: any }[])
+    const [data2, setData2] = useState({} as { [key: string]: string })
```
Comment (efef10) [inline src/components/MyAutocomplete.tsx:17]: why do we need 2 data arrays?

File: src/pages/landingPage/tabComponnent/PendingOrders.tsx [hunk 3]
```diff
@@ -1,13 +1,144 @@
...
+    const [sortBy, setSortBy] = useState(0)
+
+    const [data, setData] = useState([] as IOrder[])
+    const choose = (value: any) => {
+        setSortBy(value)
+        sort();
+
+    }
+    const sort = () => {
+        debugger
```
Comment (efef10) [inline src/pages/landingPage/tabComponnent/PendingOrders.tsx:23]: remove all debuggers and console.log

File: src/components/MyAutocomplete.tsx (hunk 2 above)
Comment (RivkaEhrenfeld) [reply src/components/MyAutocomplete.tsx:17]: mistake I changeed something and forgot to erase it

**Pull request merged**